File: Modul 1/Bahasa c/PRAK110_2310817120001_DinaIzzatiElfadheya.c
#include <stdio.h>
#include <math.h>

int main() {
    int X = 12;
    int Y = 5;
    int Z = pow (X,2)+ pow (Y,2);
    int R = sqrt(Z);
    int S = X+Y+R;
    int T = (X*Y)/2;

    printf("Diketahui : \n");
    printf("Alas = %d cm\n", Y);
    printf("Tinggi = %d cm \n\n", X);
    printf("Jawab : \n"); 
    printf("Sisi A = %d cm\n", X);
    printf("Sisi B = %d cm\n", R);
    printf("Sisi C = %d cm\n", Y);
    printf("Keliling = %d cm\n", S);
    printf("Luas = %d cm\n", T);
    return 0;

}

File: Modul 1/Bahasa Python/PRAK110_2310817120001_DinaIzzatiElfadheya.py
import math

X = 12
Y = 5
Z = (X**2)+Y**2
R = math.sqrt(Z)
S = X+Y+R
T = (X*Y)/2

print("Diketahui : ")
print("Alas =", Y, "cm")
print("Tinggi = ", X, "cm \n" )
print("Jawab : ")
print("Sisi A = ", X, "cm")
print("Sisi B = ", int(R), "cm")
print("Sisi C = ", Y, "cm")
print("Keliling =", int(S), "cm")
print("Luas = ", int (T), "cm")   